CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [6.3.0](https://github.com/theforeman/puppet-git/tree/6.3.0) (2021-04-21)
+
+[Full Changelog](https://github.com/theforeman/puppet-git/compare/6.2.0...6.3.0)
+
+**Implemented enhancements:**
+
+- Support Puppet 7 [\#74](https://github.com/theforeman/puppet-git/pull/74) ([ekohl](https://github.com/ekohl))
```

Comment:
This is a metadata only change. I'm leaning to making this 6.2.1 since as a user I don't have to think about upgrading from 6.2.0 to 6.2.1 while 6.3.0 at least makes me think a little. Thoughts?

Reply:
I think that is reasonable. The way I would do this is to keep it labeled as an 'enhancement' because in some technical sense it is one, but change the version to to 6.2.1

If you have other thoughts, please let me know, otherwise I'll assume this is what you meant

Reply:
While the code change is only a metadata change, the *intent* is an enhancement -- this will now support running with Puppet 7 and we have testing to ensure no regressions. That still feels like a .Y release to me rather than a "bug fix".

Reply:
If you follow https://semver.org/ to the letter then it would indeed be a .y. However, is metadata part of the API or implementation? It's a challenging area.

Reply:
Marking resolved since we agreed to keep this a Y release